### ISSUE TITLE
Rule description documentation

### ISSUE THREAD

**Issue by darekkay** [IBMa/equal-access#279]
## What tool(s) are you using?

- [ ] `accessibility-checker-engine`
- [x] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`

## Summary

I am creating a custom results page for the `accessibility-checker`. I would like to link to the official rule description page for each violation. I have found several references in this repository, e.g. [here](https://github.com/IBMa/equal-access/blob/887c4829e839ffcd24042636d0c55f8475dfeef8/accessibility-checker-engine/help/RPT_Html_SkipNav.mdx) is a help page for `RPT_Html_SkipNav`. Is this help page hosted anywhere? I couldn't find an official help page outside of the GitHub repository.

**Comment by tombrunet** [comment]
See https://able.ibm.com/rules/tools/help/RPT_Blockquote_WrapsTextQuote/ as an example

**Comment by darekkay** [comment]
Thank you 🙂
